terraform/alerting/alerts.tf

```diff
@@ -248,6 +248,38 @@ resource "google_monitoring_alert_policy" "UpstreamUserRecreates" {
   ]
 }
 
+resource "google_monitoring_alert_policy" "AuthenticatedSMSFailure" {
+  project      = var.project
+  combiner     = "OR"
+  display_name = "AuthenticatedSMSFailure"
+  conditions {
+    display_name = "Failed attempts to sign an SMS for authenticated SMS"
+    condition_monitoring_query_language {
+      duration = "60s"
```

Comment:
@mariliamelo @yegle can yall check my MQL here please? It's non-paging, so I don't think we need to get it 100% accurate on the first go (can tune over time), but I'd like to be at least _close_ 😄